src/components/StepperDialogWrapper/StepperDialogWrapper.jsx

```diff
@@ -1,22 +1,9 @@
 import React from 'react'
 
-import { useStepperDialogContext } from 'components/Hooks/useStepperDialogContext'
+import { useStepperDialogContext } from 'src/components/Hooks/useStepperDialogContext'
 import { useScannerI18n } from 'src/components/Hooks/useScannerI18n'
-import StepperDialog from 'components/StepperDialog/StepperDialog'
-import LazyLoad from 'components/LazyLoad/LazyLoad'
-
-const Content = () => {
-  const { allCurrentPages, currentPageIndex } = useStepperDialogContext()
-
-  return allCurrentPages.map(
-    page =>
-      page.pageIndex === currentPageIndex && (
-        <React.Fragment key={page.pageIndex}>
-          <LazyLoad currentPage={page} />
-        </React.Fragment>
-      )
-  )
-}
+import StepperDialog from 'src/components/StepperDialog/StepperDialog'
+import FormSchema from 'src/components/Form/FormSchema'
```

Comment:
nit: pour la culture, on a supprimé l'alias 'components' ?

Reply:
Dans un soucis d'homogénéité on a ajouté récemment deux alias directement dans [CS](https://github.com/cozy/create-cozy-app/blob/master/packages/cozy-scripts/config/webpack.config.base.js#L26)